### DISCUSSION TITLE
Hyphenation in EPUB

### DISCUSSION THREAD

**Discussion by zeoint** [mienaiyami/yomikiru#116] [Q&A] [answered]
What does hyphenation in EPUB reader settings do?

**Comment by mienaiyami** [comment]
Currently not working as excepted due to some unknow reason.

**Reply by mienaiyami** [reply, marked as answer]
It is supposed to make it so that word goes till end of line and break with a hyphen and other part of word comes to next line.
Hyphen is currently not visible.
